README.rst

```diff
@@ -123,3 +123,5 @@ Alternatively alternatively yapf could be set as the ``equalprg``:
 and reformat the whole file with ``gg=G`` or a single line with ``=``.
 
 .. _gq{motion}: https://github.com/vim/vim/blob/b182b40080a23ea1e1ffa28ea03b412174a236bb/runtime/doc/change.txt#L1299
+
+Note: if ``vim-yapf`` is used in this way, any errors will overwrite the current buffer. Make sure to check the buffer for errors before saving. Regardless, the ``undo`` command should work.
```

Comment:
The problem is when I type "ZZ". In that case, the vim window closes; I don't see the error; and my file contents are lost.

Is there any way to implement this without having "ZZ" lose file contents? Maybe write to a new buffer instead of overwrite current buffer? At the very least, append to current buffer?

Actually, ideal for me would be if yapf just silently failed with syntax errors.
- yapf exit code is not 0
- No vim buffers are modified

Reply:
Is it possible to print yapf syntax error to vim stdout or stderr, rather than write to a vim buffer?

Reply:
Alright, I found a workaround:

https://gist.github.com/nepsilon/003dd7cfefc20ce1e894db9c94749755

So here's what I would like to see:

Note: if ``vim-yapf`` is used in this way, any errors will overwrite the current buffer. Make sure to check the buffer for errors before saving. If you save with ``:w``, ``:undo`` will revert the buffer. If you save with ``:ZZ``, the saved file will be overwritten with the yapf syntax error. It is recommended to [Auto-backup your files](https://gist.github.com/nepsilon/003dd7cfefc20ce1e894db9c94749755) so you can recover your work.

Reply:
Note even with the workaround, I lose the most recent changes (the ones that caused error), but it's better than nothing